js/ui/ui.css

```diff
@@ -79,6 +79,15 @@
   transition: color 0.2s;
 }
 
+#fileIssueLink {
```

Comment:
nit: prefix with `respec-` and separate words with "-".

Reply:
Ideally, we only want this to show when the use selects some text. So, it would be nice to fade it in. Also, putting it down the right-hand corner is kinda ugly... might be cool to experiment with:

https://kazzkiq.github.io/balloon.css/

Reply:
> prefix with respec- and separate words with "-".

Ya I'll do this 
_______
> Ideally, we only want this to show when the use selects some text. So, it would be nice to fade it in. Also, putting it down the right-hand corner is kinda ugly... might be cool to experiment with:

Great idea. Will do this too!